packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1851,7 +1851,7 @@ export default mixins(
 					this.$store.commit('setNodeViewOffsetPosition', {newOffset: [0, 0]});
 					// For novice users (onboardingFlowEnabled == true)
 					// Inject welcome sticky note and zoom to fit
-					if(newWorkflow.onboardingFlowEnabled) {
+					if(newWorkflow.onboardingFlowEnabled && !this.isReadOnly) {
```

Comment:
can you fix spacing here
```suggestion
					if (newWorkflow.onboardingFlowEnabled && !this.isReadOnly) {
```